flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -84,6 +84,11 @@ class _SingleTabState extends State<SingleTab> {
           hwAcc: HwAcc.FULL,
           onInit: () async {
             await _controller.startRendererScanning();
+            // just a sample subtitle file
+            await _controller.addSubtitleFromNetwork(
+              'https://gist.githubusercontent.com/alr2413/9bfc7f3439ea473cbdb3ccb76d7613a3/raw/a172c10aa902078eeda9a1345c79bad9ac367d3d/subtitle_sample.srt',
+              isSelected: true,
+            );
```

Comment:
Let's not forget to remove this, too. The subtitles don't match the sample video, which might be confusing for people that are just trying out the example app.
```suggestion
```